### ISSUE TITLE
Mini preview works, but screensaver is black screen (Sonoma)

### ISSUE THREAD

**Issue by coofercat** [liquidx/webviewscreensaver#77]
This may be related to #75 - if it is, please close this one!

I'm on an M1 macbook pro, since running Sonoma (yeah I know, should have left it a few more months) all I get is a black screen from WebScreenSaver 2.2.1.

I've tried my usual screensaver (which is a pretty simple HTML + CSS + raw Javascript thing). I've also tried a "hello world" bit of plain HTML - neither work, and neither do any other websites I've tried (eg. google home page, BBC, etc). All of these look okay in Safari. I have also made sure it's not a case of black text over a black background ;-)

In System Settings -> Screensavers, I see the little preview window at the top, which "works" in so much as I see my content in it - it's all a bit wrongly scaled and whatever, but I can see it's my screensaver. If I click preview, or else trigger the screensaver on my mac, all I get is a black screen. 

I've tried uninstalling previous installs, brew installing, installing from the Zip ... you name it, they all seem to do the same thing.

Here's how it's supposed to look: https://www.pre-emptive.net/screensaver/
Here's a hacked version with all the JS removed, some HTML removed, lots of styles removed: https://www.pre-emptive.net/screensaver/simple/ (this one sometimes shows the logo, sometimes it doesn't, sometimes is just black, sometimes white)

To finish, I'd like to say how awesome this screensaver is/has been - it's a super simple and great way to 'brand' a Mac with company colours and whatnot. I've historically used it to show I'm a contractor and not one of the corporate drones in the office. I'm always amazed I don't see more people doing the same thing!

**Comment by agologan** [comment]
Definitely the same as #75
Please try out newly released `v2.3` and lmk.

**Comment by coofercat** [comment]
Awesome! v2.3 works very nicely - and I like the new preview 'eye' feature too. It feels like the settings dialogue box is more responsive too - all told, a lovely update :-)

As a side note, weirdly, even though I changed the URL from `.../screensaver/simple/` to just `.../screensaver/`, the old one continued to show when the screensaver came on (or if I click the preview box on the Settings page - not the new 'eye'). I killed off two processes called `/System/Library/Frameworks/ScreenSaver.framework/PlugIns/legacyScreenSaver.appex/Contents/MacOS/legacyScreenSaver` and then it worked fine (and the processes have been refreshed).
